src/eHealth.locations/services/selected-location-factory.js

```diff
@@ -246,8 +246,19 @@ angular.module('eHealth.locations.services')
             }
           });
           return angular.copy(adminDivisions);
+        },
+        clear: function () {
+          levels.forEach(function (level) { delete level.selected });
```

Comment:
I think that just deleting the selected attribute will break other features of a selected location. Probably in order to reset all levels playing nicely with the other features the simplest way is just calling:

    levels[0].deselect();

See [the implementation of the deselect function](https://github.com/eHealthAfrica/angular-locations/blob/master/src/eHealth.locations/services/selected-location-factory.js#L153-L157)